src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/model/reference/Cas2ApplicationStatusSeeding.kt

```diff
@@ -12,53 +12,47 @@ object Cas2ApplicationStatusSeeding {
 
   fun statusList(): List<Cas2ApplicationStatus> {
     return listOf(
-      Cas2ApplicationStatus(
-        id = UUID.fromString("c989f05f-c574-49a2-8381-11d332d98d40"),
-        name = "received",
-        label = "Received",
-        description = "The application has been received and is yet to be processed",
-      ),
       Cas2ApplicationStatus(
         id = UUID.fromString("f5cd423b-08eb-4efb-96ff-5cc6bb073905"),
         name = "moreInfoRequested",
         label = "More information requested",
-        description = "More information about the application has been requested from the POM (Prison Offender Manager).",
+              description = "The prison offender manager (POM) must provide information requested for the application to progress.",
```

Comment:
I think something has gone haywire with the indentation here, see before `description` and `label` properties....